tests/test_modeling_common.py

```diff
@@ -128,6 +128,9 @@ class ModelTesterMixin:
     test_missing_keys = True
     test_model_parallel = False
     is_encoder_decoder = False
+    # other_arg_names is a list of expected argument names in the forward pass of all variants of a model
+    # besides model.main_input_name, like ["attention_mask"] for text models
+    other_arg_names = None
```

Comment:
Can this be called something that is more explicit when reading it without the knowledge of what it is? Something like `non_main_input_names` to go with `main_input_name` ?